### DISCUSSION TITLE
github education approved but not available

### DISCUSSION THREAD

**Discussion by defenestr-hada** [community/community#142590] [GitHub Education]
### Hi everyone,

hey! I was approved github education i think on the 18th october, but a week later i still cannot access copilot, what do i have to do?
![image](https://github.com/user-attachments/assets/ec8e6038-06e3-4aa8-a2e5-6c1eab05150d)
![image](https://github.com/user-attachments/assets/a42a7e2d-b1b5-458b-b4fb-b6d31bc829aa)

**Comment by davevad93** [comment]
Hi @defenestr-hada ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @defenestr-hada was approved and your benefits are now available.`

Don't worry you don't have to do anything, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
